### PR TITLE
[KAN-18]:add info about using git in a group

### PR DESCRIPTION
**Pull request by joanwolk**
I've added some info to `README.md` and some is better than none, but it's very possible that I missed some points that should be added. 

Feel free to comment with questions we should address or anything else that comes to mind!

### CODE REVIEW

**Review by FevenSeyfu — APPROVED** [review]
Hi @joanwolk great instructions you have written. I am sure it will come in hand for most of us.

 My only suggestion is
- include commands to install and start the app, after cloning and navigating to local copy
-  if you could add tips to  prevent  conflict to begin with as it can be a rabbit hole specially for a beginner

File: README.md [hunk 1]
```diff
@@ -8,4 +8,67 @@ The challenge is to build out a multi-page website and get it looking as close t
...
+
+## Using branches
+- To check out a remote branch, such as `remotes/origin/feature/routing`, you run `git checkout feature/routing` and it will automatically set up tracking for the remote branch.
+- When you want to start working, decide whether you want to branch from `main` or from one of the other branches. There are two workflows; either is fine!
+  - Option 1:
+    - Check out the branch you want to start working from (e.g., `git checkout main` or `git checkout feature/my-feature`)
+    - Open a new branch from there: `git checkout -n feature/example` 
+  - Option 2:
+    - Open a new branch from `main`
+    - Merge the current state of any branch(es) that you want features from
```
Comment (poulami-saha) [inline README.md]:
```suggestion
    - Merge the current state of any branch(es) that you want features from using
    `git pull origin desired-branch-name`
```

File: README.md [hunk 2]
```diff
@@ -8,4 +8,67 @@ The challenge is to build out a multi-page website and get it looking as close t
...
+- "Checks" is for showing results of automated testing; we have not set this up yet.
+- "Files changed" tab shows the current diffs for all files in the PR
+  - You can leave comments on a particular line of code and it will be visible on the "Conversation" tab
+  - You can use the green "Review Changes" button to 
+    - leave general comments
+    - flag "Request changes" (a signal that something important must be done before merging)
+    - "Approve changes" (which should be done when requested changes are made to your satisfaction, or anytime everything looks mergeable without comment)
+
+## Merging PRs
+- PRs must be approved by at least one team member before they are merged.
```
Comment (poulami-saha) [inline README.md:75]:
```suggestion
- PRs must be approved by at least one team member before they are merged.
-Please do not merge your PR if you have 1 approval and there is a change requested by another approver. Please address all comments before merging the PR.
```

File: README.md [hunk 3]
```diff
@@ -8,4 +8,67 @@ The challenge is to build out a multi-page website and get it looking as close t
...
+      remotes/origin/feature/form-element-component
+      remotes/origin/feature/reusable-button-component
+      remotes/origin/feature/routing
+      remotes/origin/layout-fix
+      remotes/origin/main
+      remotes/origin/master
+      remotes/origin/responsive-footer
+      remotes/origin/responsive-footer-component
+      remotes/origin/typography-component
+    ```
```
Comment (poulami-saha) [inline README.md:34]:
```suggestion
    ```
    3. Currently please use npm package manager and not yarn for this project.
```

**Review by poulami-saha — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (joanwolk) [reply README.md:34]: Good note. Added the following as a more detailed breakdown of getting started and using npm:

```
4. This project uses npm as its package manager (not yarn). Install the dependencies by running `npm install`.
5. Start the local dev server with `npm start`.
  - If you haven't run `npm install`, you won't be able to start the site.
  - Occasionally, new dependencies may be added. When this happens, you'll probably need to run the install command again.
```

File: README.md (hunk 2 above)
Comment (joanwolk) [reply README.md:75]: Added, with markdown formatting corrected 👍

**Review by poulami-saha — APPROVED** [review]

**Pull request merged**